src/construction.cpp

```diff
@@ -947,6 +950,37 @@ void construct::done_trunk_plank( const tripoint &p )
     }
 }
 
+void construct::done_grave( const tripoint &p )
+{
+    map_stack its = g->m.i_at( p );
+    for( item it : its ) {
+        if( it.is_corpse() ) {
+            if( it.get_corpse_name().empty() ) {
+                if( it.get_mtype()->has_flag( MF_HUMAN ) ) {
+                    if( g->u.has_trait( trait_SPIRITUAL ) ) {
+                        g->u.add_morale( MORALE_FUNERAL, 50, 75, 1_days, 1_hours );
+                        add_msg( m_good, _( "You feel relieved after providing last rites for this human being, whose name is lost in The Cataclysm." ) );
```

Comment:
```suggestion
                        add_msg( m_good, _( "You feel relieved after providing last rites for this human being, whose name is lost in the Cataclysm." ) );
```
Unless I'm mistaken we don't capitalize `The`.